lighthouse-core/audits/autocomplete.js

```diff
@@ -43,6 +176,51 @@ class AutocompleteAudit extends Audit {
     };
   }
 
+  /**
+   * @param {string[]} tokenArray
+   * @return {string[]}
+   */
+  static isValidAutocompletePrefix(tokenArray) {
```

Comment:
this function and the following are non-trivial and would benefit from specific tests (using the function directly, not via `audit`).